CMakeLists.txt

```diff
@@ -124,8 +124,12 @@ endif()
 option(INSTALL_PRIVATE_HEADERS "installs private headers in the same location as the public ones" OFF)
 
 option(ENABLE_SWIFT "enable libdispatch swift overlay" OFF)
+set(SWIFT_RUNTIME_RESOURCE_INSTALL_DIR "" CACHE PATH "path to install swift compiler runtime resources")
```

Comment:
Maybe we can replace L130-132 with the line? I was hesitant about placing it into L295 because the install paths set there are identical with/without `ENABLE_SWIFT`, and `SWIFT_RUNTIME_RESOURCE_INSTALL_DIR` is just used to compute those real `INSTALL_DIR`s, which I think should be defined beforehand.